sys-cluster/teleport/files/teleport-init-d-etcd.patch

```diff
@@ -0,0 +1,13 @@
+--- work/teleport.init.d
++++ work/teleport.init.d
+@@ -4,6 +4,10 @@
+ 
+ PIDFILE=/run/teleport.pid
+ 
++depend() {
```

Comment:
I think it'd actually be better to make this configurable via a conf.d file. There's really no longer why the user just couldn't edit the config file to change the backend instead of reinstalling the whole package.